spec/ttml1.xml

```diff
@@ -13275,12 +13275,13 @@ the rate of playback (or interpretation) of time may artificially dilated or nar
 playback of a <emph>Related Media Object</emph>. Without loss of generality, the following discussion assumes a fixed play(back) rate. In the case of
 variable play rates, appropriate adjustments may need to be made to the resulting computations.</p>
 </note>
+<p>The <emph>Document Processing Context</emph> defines the applicable epoch and any epoch-related offsets to be used when establishing the <emph>Synthetic Document Syncbase</emph>.</p>
 <div2 id="time-expression-semantics-clock">
 <head>Clock Time Base</head>
 <p>When operating with the <code>clock</code> time base, the following semantics should be applied for interpreting time expressions,
 as defined by <loc href="#timing-value-timeExpression">&lt;timeExpression&gt;</loc>, and their relationship to media time and local real time.</p>
 <p>The clock time base <phrase role="strong"><code>C</code></phrase> is related to local real time <phrase role="strong"><code>R</code></phrase>
-expressed in an arbitrary (implementation defined) epoch <phrase role="strong"><code>E</code></phrase>
+expressed in the applicable epoch <phrase role="strong"><code>E</code></phrase>
```

Comment:
I would like to see the parenthetical "(implementation defined)" remain in some form.

Reply:
@skynavga Would the text suggested by @cconcolato address your comment since _Document Processing Context_ is implementation-defined?

Reply:
@palemieux yes, that would work